include/meta/meta.hpp

```diff
@@ -3162,12 +3173,15 @@ namespace meta
         /// \f$ O(log(N)) \f$.
         /// \ingroup integral
         template <typename T, T N>
-        using make_integer_sequence =
+        using make_integer_sequence = _t<
             _t<detail::coerce_indices_<T, 0, make_index_sequence<static_cast<std::size_t>(N)>>>;
```

Comment:
> apparently failing - to be funny.

I laughed as well. 

Meta did actually pretty good in the comparison that was posted on reddit the other day, are there metabench results for this yet?